_includes/downloads-scala3.html

```diff
@@ -0,0 +1,136 @@
+
+<h2>Getting Started with Scala</h2>
+<div class="install-steps">
+<p>
+  Depending on your experience and your requirements, there exist two ways to
+  install Scala 3 on your computer,
+  either through <a href="https://www.scala-sbt.org/download.html">sbt</a>, or
+  through <a href="https://get-coursier.io/docs/cli-installation">Coursier</a>.
+</p>
+
+<div class="download-options">
+  <div class="download-left">
+    <span class="or">or</span>
+    <a href="https://get-coursier.io/docs/cli-installation" class="btn-download">
+      <i class="fa fa-download"></i>
+      <span>Download Coursier</span>
+    </a>
+    <div class="text-variant">
+      <p>Best if you want to get <strong>started quickly</strong>.</p>
+      <p><i>Coursier will install all the necessary dependencies for you, so after
+         installing Coursier you are ready to go.</i></p>
+    </div>
+  </div>
+  <div class="download-right">
+    <a href="https://www.scala-sbt.org/download.html" class="btn-download">
+      <i class="fa fa-download"></i>
+      <span>Download Sbt</span>
+    </a>
+    <div class="text-variant">
+      <p>Best if you want to work on <strong>larger projects</strong>.</p>
+      <p><i>If you already have sbt installed, it is very easy to switch to Scala version {{page.release_version}}.</i></p>
+    </div>
+  </div>
+</div>
+</div>
+
+<h2>Installing Scala 3 with <strong>Coursier</strong></h2>
+<div class="install-steps">
+  <div class="step arrow">
+    <img src="/resources/img/download/arrow-asset.png" alt="">
+    <div class="number-step">1</div>
+    <div class="text-step">
+      <h3>Install the <i>Coursier</i> Scala artifact fetcher.</h3>
+      <p>Visit the <a href="https://get-coursier.io/">Coursier website</a> for detailed installation instructions on how to install Coursier.</p>
+      <p><i>(Coursier is the simplest way to get started with Scala, since it will automatically download and install all necessary dependencies for you).</i></p>
+    </div>
+  </div>
+  <!-- <div class="step">
+    <div class="number-step">2</div>
+    <div class="text-step">
+      <h3>Then, prepare your environment</h3>
+      <p>Open a terminal and run the following command: <code>cs setup</code></p>
+      <p><i>(Running this <a href="https://get-coursier.io/docs/cli-setup">command</a> installs the correct Java version along with some other helpful command line tools).</i></p>
+    </div>
+  </div> -->
+  <div class="step arrow">
+    <img src="/resources/img/download/arrow-asset.png" alt="">
+    <div class="number-step">2</div>
+    <div class="text-step">
+      <h3>Then, install Scala 3</h3>
+      <p>Open a terminal and run the following commands:</p>
+      <p><code>cs install scala3-compiler</code><span>to install the Scala 3 compiler</span></p>
+      <p><code>cs install scala3-repl</code><span>to install the Scala 3 interactive read-eval-print loop</span></p>
```

Comment:
We could skip all of this if scala 3 became the default for scala/scalac in coursier as discussed in https://github.com/coursier/coursier/issues/1992 /cc @alexarchambault

Reply:
Good point @smarter!

Reply:
I agree Scala 3 should become the default for these apps. I need to make sure the `_3` suffix is handled correctly in coursier, to enable that.

Reply:
Great! Still, I would like to change this in a separate PR. We probably also need to change all instructions for Scala 2 to deal with this changed behavior.